views/backend/write.php

```diff
@@ -33,7 +38,7 @@
                     <div class="card-content white-text">
                         <?php
                             foreach($errors as $error){
-                                echo $error."<br/>";
+                                echo $error;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of echo language construct is discouraged.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5344015)